packages/lesswrong/components/tagging/TagPageButtonRow.tsx

```diff
@@ -155,7 +176,7 @@ const TagPageButtonRow = ({tag, editing, setEditing, className, classes}: {
   </div>
 }
 
-const TagPageButtonRowComponent = registerComponent("TagPageButtonRow", TagPageButtonRow, {styles});
+const TagPageButtonRowComponent = registerComponent("TagPageButtonRow", TagPageButtonRow, { styles });
```

Comment:
I did autoformat this file... but the diff is not too bad

Reply:
I don't think small formatting errors block merging this PR, but if this file is in fact the result of an autoformatter, then that formatter is either crap or is not configured correctly for our codebase.

A few formatting problems I spotted: on lines 112-114 has `<div>` and `</div>` on different indent levels. 146-149 have open and close parens on different indent levels. Line 80 adds spaces around the `|` which we never do.

Reply:
I think Will is talking about an initial save before he noticed that his autoformatter was turned on, after which he turned it off.

Reply:
yeah I don't have eslint set up it's just the "Typescript language features" from vscode